mne/minimum_norm/tests/test_inverse.py

```diff
@@ -799,14 +796,24 @@ def test_apply_mne_inverse_epochs():
     epochs = Epochs(raw, events, event_id, tmin, tmax, picks=picks,
                     baseline=(None, 0), reject=reject, flat=flat)
 
+    return epochs
```

Comment:
you can return `Epocs(raw...` and now that you are at it, you can move as much as possible to the constructor call.

I would only keep `raw` and `picks` outside. Maybe `events` aswell. But thats it.